### DISCUSSION TITLE
How to output the tower base (0,0,0) velocity or the velocity at the center of the platform?

### DISCUSSION THREAD

**Discussion by icnosvbovwb** [OpenFAST/openfast#1714] [Q&A]
Dear @jjonkman 
I would like to know how to output tower base (0,0,0) velocity or the velocity at the center of the platform? Can you provide specific instructions on where to modify the source code output?

**Comment by jjonkman** [comment]
Dear @icnosvbovwb,

You can output the absolute platform velocity at the platform reference point via ElastoDyn output parameters `PtfmVxi`, `PtfmVyi`, `PtfmVzi` in the inertial coordinate system, or via ElastoDyn output parameters `PtfmVxt`, `PtfmVyt`, `PtfmVzt` in the tower-base coordinate system.  The location of the reference point within the platform (whether it be at the tower base, the platform center, or some other point) is user-specifed via ElastoDyn input `PtfmRefzt`.

Best regards,

**Comment by icnosvbovwb** [comment]
Dear @jjonkman 
Sorry, I don't quite understand how to operate it. For example, the top speed of the tower can be output through the variable YawBrTVxp in the outlist. However, I tried to find the variable PtfmVyt in the source code and outlist, but the message showed that I couldn't find this variable. What should I do

**Reply by jjonkman** [reply]
Oops!  It looks like I forgot to put the "T" in the output parameter names.  They should be: `PtfmTVxi`, `PtfmTVyi`, `PtfmTVzi` or `PtfmTVxt`, `PtfmTVyt`, `PtfmTVzt`.

**Comment by icnosvbovwb** [comment]
Dear @jjonkman 
Okay, thanks for your help. I have a question. For example, if I want to directly output the velocity in the y-direction of the tower base, then I can directly output PtfmTVyt, right?

**Reply by jjonkman** [reply]
Correct.
